_includes/landing-page-head.html

```diff
@@ -0,0 +1,21 @@
+<head>
```

Comment:
could you move this file into the `home` directory with the other files related to this page?

Reply:
sounds good